areas/startlocs.cpp

```diff
@@ -163,6 +164,11 @@ bool startingChoices(Player *player, std::string str, char *location, bool choos
 
         options.emplace_back("kataran");
 
+    } else if (race == KENKU) {
+
+        options.emplace_back("Tokoki Peak");
```

Comment:
pretty sure this is the area name, not display name

Reply:
I thought so too at first, but when i had it as kenku, the area option on char create was "[A] Kenku"...and when i put it as Tokoki Peak, it was [A] Tokoki Peak.